client/src/pages/private/SiteTable.js

```diff
@@ -231,7 +220,9 @@ export default ({ sites }) => {
           <Grid item>
             <Box minWidth={180}>
               <TableFilter
-                onFilter={(filteredRows) => setRows(filteredRows)}
+                onFilter={(filteredRows) => {
```

Comment:
were these brackets added intentionally?

Reply:
I can remove them. As part of my debugging I had a console.log here.